TempleEight/docs/guide/regeneration.md

```diff
@@ -3,3 +3,85 @@ id: regeneration
 title: Regeneration
 sidebar_label: Regeneration
 ---
+
+In addition to generating code from a Templefile a single time at the beginning of project development, the Temple CLI also allows for regeneration of code later in the cycle.
+This means that if you generate a project, but then want to later add an additional service, or a property on that service, Temple will allow you to regenerate code reflecting any changes to the original Templefile. 
+
+During this process, Temple will regenerate _all_ project files, overwriting any existing ones, with a few key exceptions:
```

Comment:
I wonder if we should add a comment to the top of each file about whether they’re safe to edit?